src/js/App.jsx

```diff
@@ -10,32 +9,20 @@ import Checkout from 'checkout/Checkout';
 import Confirmation from 'confirmation/Confirmation';
 import Search from 'search/Search';
 
-const App = ({
-    spots
-}) => {
-    return (
-        <Switch>
-            <Route
-                exact
-                path="/"
-                render={() => {
-                    return <Search spots={spots} />;
-                }}
-            />
-            <Route
-                path="/checkout"
-                component={Checkout}
-            />
-            <Route
-                path="/confirmation"
-                component={Confirmation}
-            />
-        </Switch>
-    );
-};
-
-App.propTypes = {
-    spots: PropTypes.arrayOf(PropTypes.object).isRequired
-};
-
-export default hot(App);
+export default hot(() => (
```

Comment:
no more establishing `App`, since I removed `spots` as a prop, this can now be simplified even more